src/test/crimson/test_alien_echo.cc

```diff
@@ -132,14 +156,14 @@ class SeastarContext {
 
   seastar::future<> set_seastar_ready() {
     // seastar: i am ready to serve!
-    ::eventfd_write(begin_fd.get(), 1);
+    ::eventfd_write(begin_fd, 1);
     return seastar::now();
   }
 
 private:
   void wait_for_seastar() {
     eventfd_t result = 0;
```

Comment:
do we need the =0 here?

Reply:
it is original code, I think better to initialize it.